src/chrome/content/rules/additifs-alimentaires.net.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="additifs-alimentaires.net">
+	<target host="additifs-alimentaires.net" />
+	<target host="www.additifs-alimentaires.net" />
+
+	<rule from="^http:"
+		to="https:" />
```

Comment:
Keep this on one line, like this: `<rule from="^http:" to="https:" />`.

Reply:
It's from the first copy/pasta of [the contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#example-ruleset-after-style-guidelines-are-applied-with-test-urls) (example ... after ... section). I'll change that.